R/dbi-driver.R

```diff
@@ -174,11 +178,13 @@ setMethod("dbConnect", "OdbcDriver",
       dbms.name = NULL,
       attributes = NULL,
       interruptible = getOption("odbc.interruptible", interactive()),
+      columnNameEncoding = "",
       .connection_string = NULL) {
     check_string(dsn, allow_null = TRUE)
     check_string(timezone, allow_null = TRUE)
     check_string(timezone_out, allow_null = TRUE)
     check_string(encoding, allow_null = TRUE)
+    check_string(columnNameEncoding, allow_null = TRUE)
```

Comment:
What does `NULL` mean here?

Reply:
Good question - i think it doesn't make sense, and changed it.
In fact, I think allowing `NULL`s for the remaining arguments that are passed to `odbc_connect` ( timezone, timeout, encoding, etc ) also doesn't make sense, as `Rcpp` won't allow that.  I'll open a chore to update those checks.

Reply:
Cleaned up NULL handling of various arguments elsewhere.